etc/flows.yaml

```diff
@@ -0,0 +1,112 @@
+---
+object_details:
+  flows:
+      execute_command:
+          uuid: "9bbc8ca2-ffda-4f18-9c7f-22b5cb6f1ef2"
```

Comment:
Is there a chance that uuid get duplicated as you are manually generating this? What is the impact if you have a duplicate?